website/src/hugo/content/changelog.md

```diff
@@ -8,13 +8,17 @@ Maintenance branches are merged before each new release. This change log is
 ordered chronologically, so each release contains all changes described below
 it.
 
-# v0.20.14 (2019-11-25)
+# v0.20.14 (2019-11-26)
 
 ## Bug fixes
 
 * [#2909](https://github.com/http4s/http4s/pull/2909): Properly propagate streamed errors in jetty-client
 * The blaze upgrade fixes the "SSL Handshake WRAP produced 0 bytes" error on JDK 11.
 
+# Enhancements
+
+* [#2911](https://github.com/http4s/http4s/pull/2911): Add missing bincompat syntax to `org.http4s.implicits`
```

Comment:
should this https://github.com/http4s/http4s/pull/2966 be mentioned here?

Reply:
That's for 0 20.14. It will be me mentioned when I add the changelog section for this release I'm seeing what else is out there so we don't have to do this a third time. Watch this space. :smile: